src/tools/storage-key-recipe-resolver.ts

```diff
@@ -126,6 +126,10 @@ export class StorageKeyRecipeResolver {
    * @param recipe long-running or ephemeral recipe
    */
   validateHandles(recipe: Recipe) {
+    recipe.handles
+      .filter(h => h.fate === 'create' && !h.capabilities.isTiedToArc && !h.id)
+      .forEach(h => console.warn(`Create handle '${h.localName}' should have an id.`));
```

Comment:
not necessarily, if the handle is persisted, but not mappable into other arcs' recipes. so i think we don't need this warning.